component/src/app/mod.rs

```diff
@@ -65,6 +67,34 @@ impl App {
         // Shielded pool always executes last.
         ShieldedPool::init_chain(&mut state_tx, app_state).await;
 
+        let mut compact_block = state_tx.stub_compact_block();
+        let mut state_commitment_tree = state_tx.stub_state_commitment_tree().await;
+
+        // Hard-coded to zero because we are in the genesis block
+        // Tendermint starts blocks at 1, so this is a "phantom" compact block
+        compact_block.height = 0;
```

Comment:
In the future, we'll want to change this to support genesis heights other than 1, and to replay historical compact blocks from the state. Maybe we should leave a note?